application/controllers/classification_type_controller.py

```diff
@@ -52,3 +52,25 @@ def get_classification_types():
         return [classification.to_dict() for classification in classification_types]
 
     logger.info("Attempted to retrieve classification types when none in database")
+
+
+def _delete_classification_query(classification_type):
+    try:
+        deleted_classification_types_count = db.session.query(ClassificationType).filter(ClassificationType.name ==
+                                                                                         classification_type).delete()
+    except SQLAlchemyError:
+        logger.exception("Unable to delete classification type", classification_type=classification_type)
+        raise DatabaseError(f'Exception thrown while trying to delete classification type {classification_type}',
```

Comment:
Maybe worth using struct log KVP for the classification type
```
raise DatabaseError('Failed to delete classification_type', classification_type=classification_type, status_code=500)
```

Reply:
The logging is key value pair? do you mean explicitly state the status_code rather than let it use the default 500?

Reply:
Sorry was getting mixed up with the wrong line